dlink-core/src/main/java/com/dlink/utils/UDFUtil.java

```diff
@@ -128,7 +132,7 @@ public static List<UDF> getUDF(String statement) {
         }
         List<String> classNameList = udfList.stream().map(UDF::getClassName).collect(Collectors.toList());
         process.info(StringUtils.join(",", classNameList));
-        process.info(StrUtil.format("A total of {} UDF have been Parsed.", classNameList.size()));
+        process.info(CharSequenceUtil.format("A total of {} UDF have been Parsed.", classNameList.size()));
```

Comment:
Why use `CharSequenceUtil` instead of `StrUtil`?

Reply:
> Why use `CharSequenceUtil` instead of `StrUtil`?

recommend using the class that the `static method` declaration is at.

Reply:
This is a good idea.